SimpleBrowser/Browser.cs

```diff
@@ -1027,7 +1099,9 @@ internal bool DoRequest(Uri uri, string method, NameValueCollection userVariable
                     
                     try
                     {
-                        using (IHttpWebResponse response = req.GetResponse())
+                        System.Threading.Thread.Sleep(100);

```

Comment:
Why does the application need to sleep here?

Reply:
Whoops, I used this in a project where the server wasn't fast enough. Should have put it in the program calling the code, not here. I am removing this.